assignments/images-challenge/code.js

```diff
@@ -23,3 +23,108 @@
  *
  *  clear()
  */
+
+const drawCircleLine = (r, color) => {
+  for (let n = 0; r * n < width - (r * 2); n = n + 2) {
+    let x = r * 2
+    let y = height / 2
+    drawFilledCircle(x + r * n, y, r, color)
+  }
+}
+drawCircleLine(10, 'red')
+
+const drawSpicyCircleLine = (y, r, color) => {
```

Comment:
I was suggesting something like this:

```suggestion
const drawSpicyCircleLine = (y, r) => {
  let color = 'red';
```

I.e. make it a local variable but not a parameter. (And then you change the call to `drawSpicyCircleLine` to just pass the y value and the radius.)